cmd/coco/reconcile/reconcile.go

```diff
@@ -204,5 +197,10 @@ var (
 		return github.New(token, owner, repo, ctx)
 	}
 	printTerminal = terminal.Output
-	readTerminal  = terminal.Read
+	readTerminal  = func() (int, error) {
+		var input int
+		_, err := fmt.Scanln(&input)
+
+		return input, err
+	}
```

Comment:
Why do you revert the usage of the terminal package?

Reply:
`scanLn` can't scan type `interface{}` which broke the whole flow of accepting input